api/src/main/java/de/oliver/fancyholograms/api/data/HologramData.java

```diff
@@ -101,8 +101,15 @@ public Visibility getVisibility() {
      * Set the type of visibility for the hologram.
      */
     public HologramData setVisibility(Visibility visibility) {
+        if (this.visibility != visibility) {
+            if (this.visibility.equals(Visibility.MANUAL)) {
```

Comment:
I guess this should be Objects.equals too then

Reply:
Visibility shouldn't be null so I think this is fine, I will add a NotNull annotation though